coffea/processor/dataframe.py

```diff
@@ -26,15 +23,23 @@ class LazyDataFrame(MutableMapping):
         flatten : bool
             Remove jagged structure from columns read
     """
-    def __init__(self, tree, entrystart=None, entrystop=None, preload_items=None, flatten=False):
-        import uproot
+
+    def __init__(
+        self, tree, entrystart=None, entrystop=None, preload_items=None, flatten=False
+    ):
         self._tree = tree
         self._flatten = flatten
-        self._branchargs = {'awkwardlib': awkward, 'flatten': flatten}
-        entrystart, entrystop = uproot.tree._normalize_entrystartstop(tree.numentries, entrystart, entrystop)
-        self._branchargs['entrystart'] = entrystart
-        self._branchargs['entrystop'] = entrystop
-        self._available = {k.decode('ascii') for k in self._tree.keys()}
+        self._branchargs = {
+            "decompression_executor": uproot4.source.futures.TrivialExecutor(),
+            "interpretation_executor": uproot4.source.futures.TrivialExecutor(),
+        }
+        if entrystart is None or entrystart < 0:
```

Comment:
Why not use https://github.com/scikit-hep/uproot4/blob/f1e66b552e930df4876c32545819918fab669cf9/uproot4/behaviors/TBranch.py#L2805-L2821?

Reply:
We have simpler logic here (no fancy negative numbers for wrapping) and that is a private function so its not guaranteed to be available/stable.

Reply:
Okay, good reasons.